tests/core/Did.spec.ts

```diff
@@ -5,11 +5,21 @@ import OperationGenerator from '../generators/OperationGenerator';
 import Encoder from '../../lib/core/versions/latest/Encoder';
 
 describe('DID', async () => {
+
+  describe('getEncodedCreateRequestFromDidString()', async () => {
+    it('should throw if the given did string is not a valid url format', () => {
+      JasmineSidetreeErrorValidator.expectSidetreeErrorToBeThrown(
+        () => (Did as any).getEncodedCreateRequestFromDidString('badURL', ['anyMethodName']),
+        ErrorCode.DidInvalidDidString
+      );
+    });
+  });
+
   describe('create()', async () => {
     it('should create a short-form DID succssefully.', async () => {
-      const expectedDidMethodName = 'did:sidetree:';
+      const expectedDidMethodName = 'sidetree';
```

Comment:
do you have a test for `sidetree:testnet` ?

Reply:
Added a testnet DID test.